src/todoList.viewModel.js

```diff
@@ -1,13 +1,21 @@
 import * as ko from 'knockout';
 import {Todo} from './todo.model.js';
 
+const DISPLAY_MODS = {
```

Comment:
DISPLAY_MODES is the right name.
See https://en.wikipedia.org/wiki/Mods_(band) for MODS